commit.go

```diff
@@ -37,5 +67,20 @@ func (b *Builder) Commit(dest types.ImageReference, options CommitOptions) error
 		return err
 	}
 	err = copy.Image(policyContext, dest, src, getCopyOptions())
+	switch dest.Transport().Name() {
+	case storage.Transport.Name():
+		tags, err := expandTags(options.AdditionalTags)
+		if err != nil {
```

Comment:
Do you want to also spin off some kind of error message for these similar to what you did above in expandTags?  Or will the returned err have enough info in it to tell which of these three conditions the error sprang from?